packages/builder/src/components/userInterface/PropertyControl.svelte

```diff
@@ -8,6 +14,45 @@
   export let props = {}
   export let onChange = () => {}
 
+  let temporaryBindableValue = value
+
+  function handleClose() {
+    handleChange(key, temporaryBindableValue)
+  }
+
+  let bindableProperties
+
+  let anchor
+  let dropdown
+
+  async function getBindableProperties() {
+    // Get all bindableProperties
+    bindableProperties = fetchBindableProperties({
+      componentInstanceId: $store.currentComponentInfo._id,
+      components: $store.components,
+      screen: $store.currentPreviewItem,
+      models: $backendUiStore.models,
+    })
+  }
+
+  async function replaceBindings(val) {
+    getBindableProperties()
+    // Find all instances of mustasche
+    const boundValues = val.match(/{{([^}]+)}}/g)
+
+    // Replace with names:
+    boundValues &&
+      boundValues.forEach(v => {
```

Comment:
NAB: Same thing with single letter variable names, and generic identifiers like `val` - it's good to be more explicit